utils/format/formatter.go

```diff
@@ -1,11 +1,14 @@
 package format
 
 import (
-	log "github.com/Sirupsen/logrus"
 	"io"
 	"os"
+
+	log "github.com/Sirupsen/logrus"
 )
 
+var osExit = os.Exit
```

Comment:
I guess this is used to avoid actual exiting during tests, but that should be documented in a comment above

Reply:
Sure.